### PR TITLE
[WIP] cli

### PR DESCRIPTION
**Pull request by pavloo**
Addresses https://github.com/pavloo/config_py/issues/1

TBD tests

### CODE REVIEW

File: bin/config_py.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    os.makedirs(conf_dir)
+    src_init_fmt = open(src_dir + '__init__.py').read()
+    dest_init_f = open(conf_dir + '__init__.py', 'w+')
+    dest_init_f.write(src_init_fmt.format(module=module))
+    shutil.copy2(src_dir + DEV_FILE, conf_dir)
+
+
+@click.command()
+@click.option("-m", "--module", help="module to create a config for", default=None)
```
Comment (pavloo) [inline bin/config_py.py]: change help to "*python* module to create a config for"

File: bin/config_py.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import shutil
+
+
+CONF_DIR_NAME = 'config'
+DEV_FILE = 'config-dev.py'
+ERROR_ALREADY_EXISTS_FMT = '"config" module already exists for the {} module'
+
+
+def generate_root_config():
+    src_dir = sys.path[0] + '/../fixtures/root/'
```
Comment (pzmudzinski) [inline bin/config_py.py]: It should be all done via `os.path.join` - we want to run on Windows, don't we/

Comment (pavloo) [reply bin/config_py.py]: good call

File: bin/config_py.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    if os.path.exists(conf_dir):
+        click.secho(
+            ERROR_ALREADY_EXISTS_FMT.format('"{}"'.format(module)),
+            err=True,
+            fg='red'
+        )
+        return
+
+    os.makedirs(conf_dir)
+    src_init_fmt = open(src_dir + '__init__.py').read()
```
Comment (pavloo) [inline bin/config_py.py]: don't forget to close files, prolly use context `open`

**Comment by pavloo** [comment]
@pzmudzinski added tests, cleaned up everything. ready for review

**Pull request merged**